### PR TITLE
ENH: 2023 conference metadata

### PR DESCRIPTION
**Pull request by deniederhut**
Adds publishing metadata from the 2023 conference

### CODE REVIEW

File: publisher/metadata/2023/scipy_proc.json [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+          "org": "USGS",
+          "name": "Mike Hearne"
+        }
+      ]
+    },
+    {
+      "name": "Proceedings",
+      "members": [
+        {
+          "org": "",
```
Comment (mepa) [inline publisher/metadata/2023/scipy_proc.json]:
```suggestion
          "org": "GDI",
```

**Review by mepa — COMMENTED** [review]
Do these metadata need to be in sync with what's in the 2023 branch (e.g., https://github.com/scipy-conference/scipy_proceedings/pull/878)?

**Comment by deniederhut** [comment]
Good catch!

**Comment by rowanc1** [comment]
We have moved this branch/pull requests over to here:
https://github.com/scipy-conference/scipy_proceedings_archive/tree/main/scipy2023/proceedings

**Pull request closed without merging**